data/json/recipes/recipe_food.json

```diff
@@ -4436,7 +4438,6 @@
     [
       [ "garlic", 1 ],
       [ "seed_garlic", 6 ],
-      [ "thyme", 2 ],
```

Comment:
That thyme wasn't wrong, garlic can be supplied both in crafted and uncrafted forms, just that it should be 1 of it rather than 2.